configure.ac

```diff
@@ -2977,7 +2977,7 @@ then
                         AM_CPPFLAGS="$AM_CPPFLAGS+sm4"
                     fi
                 else
-                    AM_CPPFLAGS="$AM_CPPFLAGS -mcpu=generic+crypto"
+                    AM_CPPFLAGS="$AM_CPPFLAGS -march=armv8.1-a+crypto"
```

Comment:
Customers target is not v8.1. Needs to be v8.0 `-march=armv8-a+crypto`, however that does not allow `sqrdmlsh`. Is there a way to not use `sqrdmlsh`?

```
 CC       wolfcrypt/src/src_libwolfssl_la-dilithium.lo
wolfcrypt/src/port/arm/armv8-kyber-asm.S: Assembler messages:
wolfcrypt/src/port/arm/armv8-kyber-asm.S:266: Error: selected processor does not support `sqrdmlsh v21.8h,v29.8h,v4.h[0]'
wolfcrypt/src/port/arm/armv8-kyber-asm.S:267: Error: selected processor does not support `sqrdmlsh v22.8h,v30.8h,v4.h[0]'
wolfcrypt/src/port/arm/armv8-kyber-asm.S:274: Error: selected processor does not support `sqrdmlsh v23.8h,v29.8h,v4.h[0]'
wolfcrypt/src/port/arm/armv8-kyber-asm.S:275: Error: selected processor does not support `sqrdmlsh v24.8h,v30.8h,v4.h[0]'
wolfcrypt/src/port/arm/armv8-kyber-asm.S:282: Error: selected processor does not support `sqrdmlsh v25.8h,v29.8h,v4.h[0]'
wolfcrypt/src/port/arm/armv8-kyber-asm.S:283: Error: selected processor does not support `sqrdmlsh v26.8h,v30.8h,v4.h[0]'
wolfcrypt/src/port/arm/armv8-kyber-asm.S:290: Error: selected processor does not support `sqrdmlsh v27.8h,v29.8h,v4.h[0]'
wolfcrypt/src/port/arm/armv8-kyber-asm.S:291: Error: selected processor does not support `sqrdmlsh v28.8h,v30.8h,v4.h[0]'
```

FYI: When I run on ZCU102 A53 I get:

```
./configure --host=aarch64 CC="aarch64-linux-gnu-gcc" AR="aarch64-linux-gnu-ar" RANLIB="aarch64-linux-gnu-ranlib" --enable-sp=yes,asm --enable-keygen --enable-armasm --enable-experimental --enable-kyber --enable-dilithium --enable-keygen --enable-lms --enable-xmss --enable-curve25519 --enable-ed25519 --enable-curve448 --enable-ed448 --disable-shared --enable-static --disable-dh --disable-filesystem && make
```

```
./benchmark -kyber
------------------------------------------------------------------------------
 wolfSSL version 5.7.2
------------------------------------------------------------------------------
Math:   Multi-Precision: Wolf(SP) word-size=64 bits=4096 sp_int.c
        Single Precision: ecc 256 384 521 rsa 2048 3072 4096 asm sp_arm64.c
wolfCrypt Benchmark (block bytes 1048576, min 1.0 sec each)
Illegal instruction
```

```
CURVE448 test passed!
ED448    test passed!
Illegal instruction
```